src/Evaluators/bridge_evaluator.jl

```diff
@@ -0,0 +1,138 @@
+
+#=
+    BridgeEvaluator
+=#
```

Comment:
I am still not sure what the role of the `BridgeEvaluator` is. It uses a model on a device (e.g. `PolarForm`) and returns the output data of the evaluation functions on the selected device?